### ISSUE TITLE
Accessibility - Missing Form Label on Log Viewer for the 'Time Period' field

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#13250]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

8.18.2

### Bug summary

When I run the 'Wave' accessibility checker tool from 'Webaim' on the Log Viewer page I am advised that there is an Error.

The Error relates to the 'Time Period' field where the user specifies which dates they wish to search the event logs for.

ISSUE DETAILS :
**What It Means**
A form control does not have a corresponding label.

**Why It Matters**
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.

**How to Fix It**
If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

### Steps to reproduce

To replicate the issue do the following:

1. Login to the CMS
2. Access the Log Viewer page
3. Perform a check of the page using the webaim Wave accessibility checker tool
4. Study results for errors

### Expected result / actual result

There will be no error returned when the Wave Accessibility testing tool is run on the page.
